packages/an-controller/contracts/ANController.sol

```diff
@@ -0,0 +1,47 @@
+pragma solidity 0.5.17;
```

Comment:
We should enable syntax highlighting in `.gitattributes`

Reply:
Fixed in 88499a4